gallery-demo/build.gradle.kts

```diff
@@ -0,0 +1,122 @@
+import org.jetbrains.compose.reload.ComposeHotRun
```

Comment:
I have a small question related to this module. We have the `example` one, which contains an example of the demo. Was it intentional to create a new one instead of re-using the `example`?

Reply:
the existing `example` requires a publication of all the modules to mavenLocal. 
It was more convenient to iterate and test the gallery changes without a publication. 

The existing `example` module is still useful to test the integration of all modules - we have CI checks with it. 
We have something similar is Compose Multiplatform too: a demo right in the core project - it uses project dependencies, and a set of side projects which integrate everyting as published module dependencies.